sample/github/legit.go

```diff
@@ -50,15 +49,23 @@ type GithubSecrets struct {
 	SecretToken string `json:"secretToken"`
 }
 
-// HandlePullRequest is invoked whenever a PullRequest is modified (created, updated, etc.)
-func (handler *GithubHandler) HandlePullRequest(payload interface{}, header webhooks.Header) {
-	glog.Info("Handling Pull Request")
+func (h *GithubHandler) handlePost(rw http.ResponseWriter, req *http.Request) {
```

Comment:
If you use `pkg/events`, it should reduce the boilerplate in 52-64 to about 2 lines.

Reply:
events assumed CloudEvent and I want to do that but not in this PR.

Reply:
https://github.com/knative/eventing/issues/334